src/specs/neon1.yaml

```diff
@@ -0,0 +1,178 @@
+openapi: 3.0.3
+info:
+  title: Neon Specific API methods
+  description: |-
+    This is a specification of the Neon API methods written up as if it is a REST API that is compliant with the OpenAPI 3.0 specification. This is a hack to use the tooling available to render the spec as HTML. The intention is to imitate this tooling for our own purposes and remove the implication that this is a REST API.
+
+    The source of truth is [this Notion page](https://www.notion.so/neonlabs/Neon-Specific-API-methods-3402baaad8fa4daeb12642495cf85eb3)
+  termsOfService: wip
+  contact:
+    email: info@neonevm.org
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.11
+externalDocs:
+  description: Find out more about Neon EVM
+  url: https://docs.neonfoundation.io/docs/quick_start
+servers:
+  - url: https://wip
+tags:
+  - name: Settings
+    description: This family of endpoints return production versions and EVM parameters.
+    # externalDocs:
+    #   description: IF docs provides a support page, can target that directly
+    #   url: https://docs.neonfoundation.io/docs/quick_start
+  - name: Transactions
+    description: This family of endpoints provide access to transaction-related data.
+ # externalDocs:
+    #   description: IF docs provides a support page, can target that directly
+    #   url: https://docs.neonfoundation.io/docs/quick_start
+paths:
```

Comment:
@afalaleev This doc presents your Neon-specific api calls as if they were Rest calls as per Open API 3+ which they are not (!)
If you paste this file into https://editor.swagger.io you can see what the UX will be like
However, there are many assumptions taken here -- the highest risk item is that there are individual endpoints to hit -- as the schema lays out the paths (see hierarchy at line 31) HOWEVER, I can see that you are passing a method which means you could have 1 endpoint exposed that is providing the service to all the methods you described -- this is not a rest-native approach and I am not sure I could have open API to suit if this is the case